### PR TITLE
add resource component to fix repository arn

### LINKED ISSUE #715: Python Lambda From Container ARN error
### Describe the bug

Setting `use_pre_existing_image = True` causes the cdk synth process to fail.

> jsii.errors.JSIIError: The `resource` component (6th component) of an ARN is required: arn:aws:ecr:${Token[AWS.Region.7]}:${Token[AWS.AccountId.3]}

### Expected Behavior

The cdk synth process should finish successfully.

### Current Behavior

Setting `use_pre_existing_image = True` causes the cdk synth process to fail.

> jsii.errors.JSIIError: The `resource` component (6th component) of an ARN is required: arn:aws:ecr:${Token[AWS.Region.7]}:${Token[AWS.AccountId.3]}

### Reproduction Steps

Set `use_pre_existing_image = True` in the app.py in the `python/lambda-from-container` example

### Possible Solution

Adding the resource component to the ARN string should resolve this issue.

### Additional Information/Context

_No response_

### CDK CLI Version

2.37.1 (build f15dee0)

### Framework Version

_No response_

### Node.js Version

v18.7.0

### OS

Mac OSX

### Language

Python

### Language Version

Python 3.8.9

### PR DESCRIPTION
**Pull request by jonathan-eckel**
fix the repository arn

Fixes #715 
---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: python/lambda-from-container/app.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
             ##
             ecr_repository = aws_ecr.Repository.from_repository_attributes(self,
                 id              = "ECR",
-                repository_arn  ='arn:aws:ecr:{0}:{1}'.format(Aws.REGION, Aws.ACCOUNT_ID),
+                repository_arn  ='arn:aws:ecr:{0}:{1}:repository'.format(Aws.REGION, Aws.ACCOUNT_ID),
```
Comment (peterwoodworth) [inline python/lambda-from-container/app.py:38]: This isn't the correct format still - Repository arns require the slash name component https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticcontainerregistry.html#amazonelasticcontainerregistry-resources-for-iam-policies

**Review by peterwoodworth — CHANGES_REQUESTED** [review]
Thank you very much for the contribution, and sorry for the delay in response

Have you tested this? The arn format still isn't what is expected of a repository arn. We might need the user to submit their repository name as an input for this example if there is no other workaround

**Comment by jonathan-eckel** [comment]
This PR is from over 6 months ago.  I don't work with awscdk anymore, so I won't be updating this PR or issue.

File: python/lambda-from-container/app.py (hunk 1 above)
Comment (stevebanik) [reply python/lambda-from-container/app.py:38]: @peterwoodworth I opened https://github.com/aws-samples/aws-cdk-examples/issues/821 as I am seeing the same issue

**Pull request closed without merging**